src/agentscope/models/dashscope_model.py

```diff
@@ -505,18 +505,10 @@ def __call__(
         )
 
         # step5: return response
-        if len(response.output["embeddings"]) == 0:
-            return ModelResponse(
-                embedding=response.output["embedding"][0],
-                raw=response,
-            )
-        else:
-            return ModelResponse(
-                embedding=[
-                    _["embedding"] for _ in response.output["embeddings"]
-                ],
-                raw=response,
-            )
+        return ModelResponse(
```

Comment:
This modification provides unified interface for embedding vectors, and it's much better than previous version.